src/v3/src/components/Widget/index.tsx

```diff
@@ -71,16 +73,18 @@ import {
   triggerEmailVerifyCallback,
 } from '../../util';
 import { getEventContext } from '../../util/getEventContext';
-import { mapMuiThemeFromBrand } from '../../util/theme';
+import { createTheme } from '../../util/theme';
 import AuthContainer from '../AuthContainer/AuthContainer';
 import AuthContent from '../AuthContent/AuthContent';
 import AuthHeader from '../AuthHeader/AuthHeader';
-import ConsentHeader from '../ConsentHeader';
 import CustomPluginsOdysseyCacheProvider from '../CustomPluginsOdysseyCacheProvider';
 import Form from '../Form';
 import IdentifierContainer from '../IdentifierContainer';
 import Spinner from '../Spinner';
 import GlobalStyles from './GlobalStyles';
+import Button from '../Button';
+import { ScopedCssBaseline } from '@mui/material';
```

Comment:
linting should catch this so lmk if it's not but this import should be at the top with other external imports